stable/cluster-autoscaler/values.yaml

```diff
@@ -191,6 +191,7 @@ serviceMonitor:
    ## [Kube Prometheus Selector Label](https://github.com/helm/charts/tree/master/stable/prometheus-operator#exporters)
   selector:
     prometheus: kube-prometheus
+    release: prometheus-kubeaddons
```

Comment:
This should be in the addon, not the chart default values.

Reply:
i think that is arguable. There are enough konvoy specific items i dont see that as an issue, but yeah maybe those should be revisited.

Reply:
@joejulian are u blocking a PR for having a label with kubeaddons? This is a big contradiction if you consider the rest of charts and their default values.

Reply:
It's not a label, it's a selector. By changing this to something d2iq specific when it can just as easily be overridden in the addon, we're diverging even further from upstream and making this chart harder to maintain. These kind of changes should be done in the addon record.

Reply:
Wait... My bad. I read "prometheus" and missed that this wasn't prometheus and is rather our non-upstream chart. I'm sorry for that misunderstanding. I'm hitting approve.